lib/Multihash.ts

```diff
@@ -79,18 +79,23 @@ export default class Multihash {
   }
 
   /**
-   * Checks if the given hash is a multihash computed using the configured hashing algorithm.
+   * Checks if the given encoded hash is a multihash computed using the configured hashing algorithm.
    */
-  public static validateHashComputedUsingSupportedHashAlgorithm (
+  public static validateEncodedHashComputedUsingSupportedHashAlgorithm (
     encodedMultihash: string,
     inputContextForErrorLogging: string
   ) {
+    if (!Encoder.isBase64UrlString(encodedMultihash)) {
+      throw new IonError(ErrorCode.EncodedMultiHashIncorrectEncoding, `Given ${inputContextForErrorLogging} must be base64url string.`);
+    }
```

Comment:
Sanity: this is optional as surely line 92 will throw if this isn't base64url encoded? Is this just so that we very nice error message? If so maybe move it into `decodeAsBuffer()` itself?